packages/xod-cli/README.md

```diff
@@ -9,23 +9,245 @@ other packages. The responsibility of `xod-cli` is to parse command line
+* [`xodc tabtest [options] [entrypoint]`](#xodc-tabtest-options-entrypoint)
+* [`xodc transpile [options] [entrypoint]`](#xodc-transpile-options-entrypoint)
+
+## `xodc autocomplete [SHELL]`
+
+display autocomplete installation instructions
+
+```
+USAGE
+  $ xodc autocomplete [SHELL]
+
+ARGUMENTS
+  SHELL  shell type
+
+OPTIONS
+  -r, --refresh-cache  Refresh cache (ignores displaying instructions)
+
+EXAMPLES
+  $ xodc autocomplete
+  $ xodc autocomplete bash
+  $ xodc autocomplete zsh
+  $ xodc autocomplete --refresh-cache
+```
+
+_See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomplete/blob/v0.1.0/src/commands/autocomplete/index.ts)_
+
+## `xodc help [COMMAND]`
+
+display help for xodc
+
+```
+USAGE
+  $ xodc help [COMMAND]
+
+ARGUMENTS
+  COMMAND  command to show help for
+
+OPTIONS
+  --all  see all commands in CLI
+```
+
+_See code: [@oclif/plugin-help](https://github.com/oclif/plugin-help/blob/v2.1.3/src/commands/help.ts)_
+
+## `xodc publish [options] [project]`
+
+publish a library
+
+```
+USAGE
+  $ xodc publish [options] [project]
+
+ARGUMENTS
+  PROJECT
+      Project to operate on. The project should point to a file or directory
+      on file system. If omitted, it is inferred from the current working
+      directory. Examples:
+
+         * ./path/to/proj.xodball           # xodball
+         * ./path/to/proj                   # just full path to a project
+
+OPTIONS
+  -V, --version          show CLI version
+  -h, --help             show CLI help
+  -q, --quiet            do not log messages other than errors
+  --api=api              [default: xod.io] XOD API hostname (default: xod.io)
+  --on-behalf=on-behalf  publish on behalf of the username
+  --password=password    XOD API password
+  --username=username    XOD API username
+
+EXAMPLES
+  Publish the current project with the version defined in `project.xod`
+  $ xodc publish
+  Publish a project saved as xodball
+  $ xodc publish foo.xodball
+```
+
+_See code: [src/commands/publish.js](https://github.com/xodio/xod/blob/master/packages/xod-cli/src/commands/publish.js)_
+
+## `xodc resave [options] [project]`
+
+opens a project and saves it in another location or format
+
+```
+USAGE
+  $ xodc resave [options] [project]
+
+ARGUMENTS
+  PROJECT
+      Project to operate on. The project should point to a file or directory
+      on file system. If omitted, it is inferred from the current working
+      directory. Examples:
+
+         * ./path/to/proj.xodball           # xodball
+         * ./path/to/proj                   # just full path to a project
+
+OPTIONS
+  -V, --version              show CLI version
+  -h, --help                 show CLI help
+  -o, --output=output        xodball or multifile directory output path, defaults to stdout
+  -q, --quiet                do not log messages other than errors
+  -w, --workspace=workspace  [default: /home/sk/xod] use the workspace specified, defaults to $HOME/xod
```

Comment:
The same:

```
-w, --workspace=path
```

And can the default render as `~/xod`, not `/home/whoever-updated-the-readme/xod`?